app/models/test_group.rb

```diff
@@ -0,0 +1,56 @@
+class TestGroup < ApplicationRecord
+  TO_INSTRUCTORS = 'instructors'.freeze
```

Comment:
Would be nice to use enums. https://api.rubyonrails.org/v5.2.2/classes/ActiveRecord/Enum.html

Reply:
Cool, I'll try them